docs/index.md

```diff
@@ -1,19 +1,95 @@
-# What is rpm-ostree?
+---
```

Comment:
Hmm, is there no way to make the `README.md` and the `index.md` the same page? Or if there is, is there a reason why we don't want to do that?

Reply:
We can keep the same content in both pages but that requires manual sync as we can not use symlinks in either direction. I can make that happen if that's your preference.

Reply:
I updated the readme to match the index page.

Reply:
Ahh gotcha, oh well that's too bad. The duplication is not great, though we can address it in a follow-up!